docs/source/quickstart.md

```diff
@@ -8,7 +8,7 @@ The library was designed with two strong goals in mind:
 
 - be as easy and fast to use as possible:
 
-  - we strongly limited the number of user-facing abstractions to learn, in fact there are almost no abstractions, just three standard classes required to use each model: configuration, models and tokenizer,
+  - we strongly limited the number of user-facing abstractions to learn, in fact, there are almost no abstractions, just three standard classes required to use each model: configuration, models and tokenizer,
```

Comment:
```suggestion
  - we strongly limited the number of user-facing abstractions to learn. In fact, there are almost no abstractions, just three standard classes required to use each model: configuration, models and tokenizer.
```

Reply:
All three sub-points (a) start with small caps, and (b) didn't end with a full-stop. So, I thought not to the sentence. Anyway, IMHO, breaking too long sentences is always a good idea.